packages/hardhat-etherscan/test/unit/ChainConfig.ts

```diff
@@ -3,7 +3,7 @@ import { chainConfig } from "../../src/ChainConfig";
 
 describe("Chain Config", () => {
   it("should have no duplicate chain ids", () => {
-    const chainIds = Object.values(chainConfig);
+    const chainIds = Object.values(chainConfig).map((config) => config.chainId);
```

Comment:
I think this was an error. The way it was before, this test would always pass.

Reply:
Well spotted. I think I refactored the structure of chainConfig, and the test worked before and after.

I have added a type guard on chainIds (number[]) to pickup changes in the structure of chainConfig.